src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,7 +58,7 @@ public PersonCard(Person person, int displayedIndex) {
         nokPhone.setText("TODO");
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
-        classDetails.setText("TODO");
+        classDetails.setText(person.getAClass().classToString);
```

Comment:
As mentioned in the comment in `Class.java`, it might be better to make use of `toString()` method instead of value, as this section differs from other fields which can directly make use of String values just as stored.